depthai_nodes/ml/parsers/yolo.py

```diff
@@ -1,11 +1,21 @@
+from typing import Any, Dict, List, Optional
```

Comment:
all functions need type hints and return types. See setOutputLayerNames()

Reply:
OK, updating all the type hints and return types in functions with the next commit.